Tools/LyTestTools/ly_test_tools/environment/file_system.py

```diff
@@ -456,6 +456,7 @@ def find_ancestor_file(target_file_name, start_path=os.getcwd()):
             current_path = parent_path
         else:
             # Found the file we wanted
+            candidate_path = os.path.abspath(candidate_path)
```

Comment:
Ah so the issue was that build directory was relative?

So a relative build directory from the engine root of of `AutomatedTesting/build/windows_vs2019/bin/profile` would return `AutomatedTesting` instead, which would then get concatenated to the current working directory using the python os.path.join rules.

I think we should make the current_path absolute instead of using normpath

Reply:
ah, yeah that would be cleaner. fixed.